sda-commons-server-cloudevents/src/main/java/org/sdase/commons/server/cloudevents/CloudEventV1.java

```diff
@@ -0,0 +1,181 @@
+package org.sdase.commons.server.cloudevents;
+
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import java.net.URI;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+/**
+ * @param <T> the type of the data element of this CloudEvent
+ * @see <a href="https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md">1.0.2
+ *     Specification</a>
+ */
+public class CloudEventV1<T> extends BaseCloudEvent {
+
+  @JsonPropertyDescription(
+      "Identifies the event. Producers MUST ensure that `source` + `id` is "
+          + "unique for each distinct event. If a duplicate event is re-sent (e.g. due to a network "
+          + "error) it MAY have the same id. Consumers MAY assume that Events with identical `source` "
+          + "and `id` are duplicates.")
+  @NotEmpty
+  private String id = UUID.randomUUID().toString();
+
+  @JsonPropertyDescription(
+      "Identifies the context in which an event happened. Often this will "
+          + "include information such as the type of the event source, the organization publishing the "
+          + "event or the process that produced the event. The exact syntax and semantics behind the "
+          + "data encoded in the URI is defined by the event producer.\n"
+          + "\n"
+          + "Producers MUST ensure that `source` + `id` is unique for each distinct event.\n"
+          + "\n"
+          + "An application MAY assign a unique `source` to each distinct producer, which makes it "
+          + "easy to produce unique IDs since no other producer will have the same source. The "
+          + "application MAY use UUIDs, URNs, DNS authorities or an application-specific scheme to "
+          + "create unique source identifiers.\n"
+          + "\n"
+          + "A source MAY include more than one producer. In that case the producers MUST collaborate "
+          + "to ensure that `source` + `id` is unique for each distinct event.")
+  @NotNull
+  private URI source;
```

Comment:
the "source" is for a specific service always the same. How can we provide documentation "hooks" in order to reflect this?